content/kit/power-board.md

```diff
@@ -2,36 +2,36 @@
 title: Power Board
 ---
 
-The Power Board distributes power to the SR kit from the battery. It provides
-six individual general-purpose power outputs along with a separate power connector
-for the Brain Board.
+The Power Board distributes power from the battery to the rest of the kit. It provides
+six individual general-purpose 12V power outputs along with a separate power connector
+for the Raspberry Pi and servos.
 
 It also holds the internal On|Off switch for the whole robot as well as
-the Start button which is used to start your robot code running.
+the Start button which is used to start your robot code.
 
-## Board Diagram
+## Board diagram
 ![Power Board Diagram](/img/kit/power_board_v4_diagram.png)
 
 ## Connectors
-There are six power output connectors on the board, labelled L0–L3, H0, and H1. These are enabled when your robot code is started, and supply around 11.1V (±15%). They should be used to connect to the motor and servo board power inputs. The "H" connectors will supply more current than the "L" connectors.
+There are six power output connectors on the board, labelled L0–L3, H0, and H1. These are enabled when your robot code is started, and supply around 11.1V (±15%). They should be used to connect to the motor board power input. The "H" connectors will supply more current than the "L" connectors.
 
-The 5V connectors can be used to connect low-current devices that take 5V inputs, such as the Brain Board.
+The 5V connectors can be used to connect low-current devices that take 5V inputs, such as the Raspberry Pi and the servo shield.
 
-There is also a Micro USB B connector which should be used to connect the Brain Board for control of the power board.
+There is also a Micro USB B connector which should be used to connect the Raspberry Pi for control of the power board.
 
-Finally, there are connectors for external Start and On\|Off switches. You may connect any push-to-make latching switch for the On\|Off button, or a push-to-make button for the start button.
+Finally, there are connectors for external Start and On|Off switches. You may connect any latching switch for the On|Off switch, or a push-to-make button for the Start button.
 
 {{% notice note %}}
-If you intend to use only the internal On|Off switch, you must connect a CamCon to the On|Off connector with a wire connecting one pin to the other pin on the same connector.
+If you intend to use only the internal On|Off switch, you must plug a CamCon into the On|Off connector with a wire connecting one pin to the other pin on the same connector.
```

Comment:
It's probably worth stating here that the kit we provide them already has one attached

Reply:
Good idea, thanks.